umm-spec-lib/src/cmr/umm_spec/umm_g/related_url.clj

```diff
@@ -20,7 +20,7 @@
       :size Size})))
 
 (defn umm-g-related-urls->RelatedURLs
-  "Returns the umm-lib granule model RelatedURLs from the given UMM-G RelatedUrls."
+  "Returns the umm-lib granule model RelatedURLs from the given UMM-G RelatedUrls. note: this is new->old"
```

Comment:
I don't understand why you're adding the notes of new->old and old->new. Everything needs to be parsed through umm-lib. They are all the same thing just in different format.

Reply:
I could and maybe should add more text there, but as Erich was walking me though the code I found it important to write down that this function is converting data from the old structure to the new structure since there was an effort to rewrite some of the umm-g code.

Reply:
is this better:
  "Returns the umm-lib granule model RelatedURLs from the given UMM-G RelatedUrls.
  NOTE: this is converts from the new style of urls to the old style."